libbeat/docs/version.asciidoc

```diff
@@ -1,7 +1,7 @@
 :stack-version: 7.0.0-alpha1
 :doc-branch: master
-:go-version: 1.10.3
-:release-state: prerelease
+:go-version: 1.11.2
+:release-state: unreleased
```

Comment:
This worries me. Why did this change? This line should be reverted.

Reply:
I guess rebase did opt for the old one. It must say `prerelease`